Content.Server/Parallax/BiomeSystem.Commands.cs

```diff
@@ -151,14 +153,30 @@ private void AddMarkerLayerCallback(IConsoleShell shell, string argstr, string[]
             return;
         }
 
-        biome.MarkerLayers.Add(args[1]);
+        if (!biome.MarkerLayers.Add(args[1]))
+        {
+            return;
+        }
+
+        biome.ForcedMarkerLayers.Add(args[1]);
     }
 
     private CompletionResult AddMarkerLayerCallbackHelper(IConsoleShell shell, string[] args)
     {
         if (args.Length == 1)
         {
-            return CompletionResult.FromHintOptions(CompletionHelper.Components<BiomeComponent>(args[0], EntityManager), "Biome");
+            var allQuery = AllEntityQuery<MapComponent>();
```

Comment:
Can't this query for both MapComponent and BiomeComponent at once?

Reply:
fuck